client/src/extension.ts

```diff
@@ -48,10 +49,40 @@ export function activate(context: ExtensionContext) {
 		}
 	};
 
+	workspace.getConfiguration().update('yaml.customTags', [
+		'!And',
+		'!If',
+		'!Not',
+		'!Equals',
+		'!Or',
+		'!FindInMap',
+		'!Base64',
+		'!Cidr',
+		'!Ref',
+		'!Sub',
+		'!GetAtt',
+		'!GetAZs',
+		'!ImportValue',
+		'!Select',
+		'!Split',
+		'!Join'
+	], ConfigurationTarget.Global);
+	workspace.getConfiguration().update('[yaml]', {
```

Comment:
Not sure we should set these additional settings.  It may have to be some sore of merge but I'm not sure how hard it will be to see if singular settings are changed from the default.